pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1541,4 +1541,15 @@ self: super: {
   # https://github.com/yesodweb/yesod/issues/1714
   yesod-core = dontCheck super.yesod-core;
 
+  # Add ApplicationServices on darwin
+  # Patch can be removed if https://github.com/jonascarpay/apecs/pull/77 is resolved
+  apecs-physics =
+    let
+      sysDeps = pkgs.lib.optional pkgs.stdenv.isDarwin pkgs.darwin.apple_sdk.frameworks.ApplicationServices;
+    in addPkgconfigDepends (appendPatch super.apecs-physics (pkgs.fetchpatch {
+      url = "https://github.com/jonascarpay/apecs/pull/77/commits/24975a4e689fb9920168bbbcb451d8fb8a16abdb.patch";
+      sha256 = "0w7ivv9wgrkba9psbc4920y0apg5cpyz9fwwzizfgqhy0gz6ss7y";
+      stripLen = 1;
+    })) sysDeps;
```

Comment:
Ping @jonascarpay.

Does this change look okay to you, or does it make more sense just to wait for you to release a new version of `apecs-physics` to Hackage?